packages/blockchain-extension/extension/commands/UserInputUtil.ts

```diff
@@ -1279,16 +1279,18 @@ export class UserInputUtil {
         }
     }
 
-    private static async getInstalledContracts(connection: IFabricEnvironmentConnection, peers: Array<string>): Promise<IBlockchainQuickPickItem<{ packageEntry: PackageRegistryEntry, workspace: vscode.WorkspaceFolder }>[]> {
+    // Only being used for v1 contracts
+    private static async getInstalledContracts(connection: IFabricEnvironmentConnection, peers: Array<string>, isV1: boolean = false): Promise<IBlockchainQuickPickItem<{ packageEntry: PackageRegistryEntry, workspace: vscode.WorkspaceFolder }>[]> {
         const tempQuickPickItems: IBlockchainQuickPickItem<{ packageEntry: PackageRegistryEntry, workspace: vscode.WorkspaceFolder }>[] = [];
         for (const peer of peers) {
-            const chaincodes: FabricInstalledSmartContract[] = await connection.getInstalledSmartContracts(peer);
+            const chaincodes: FabricInstalledSmartContract[] = await connection.getInstalledSmartContracts(peer, isV1);
 
             // TODO: this is wrong but won't be needed later
             chaincodes.forEach((chaincode: FabricInstalledSmartContract) => {
-                const _package: PackageRegistryEntry = new PackageRegistryEntry({ name: chaincode.label, version: chaincode.packageId, path: undefined, sizeKB: undefined });
+                const nameAndVersion: string[] = chaincode.label.split('@');
```

Comment:
I _think_ in our v1 extension we allow a user to instantiate a contract with a package such as `packageWithoutVersion.cds` - might be worth double-checking that and confirming that this code will work okay if that's the case.